.mk/k8s.mk

```diff
@@ -56,7 +56,7 @@ SPIRE_ENABLED=false
 endif
 
 kubectl = kubectl -n ${NSM_NAMESPACE}
-images_tar = $(subst .tar,,$(filter %.tar, $(shell ls ./scripts/vagrant/images)))
+images_tar = $(subst .tar,,$(filter %.tar, $(shell mkdir -p ./scripts/vagrant/images;ls ./scripts/vagrant/images)))
```

Comment:
Actually I think we need to move from using of ./scripts/**vagrant** folder as a place for docker images

Reply:
> we need to move from using of ./scripts/vagrant folder as a place for docker images

+1
We might also want a variable in build.mk for this, like IMAGE_DIR

Reply:
Yeah... we need a better way here... I took the path of the minimum fix that actually fixed the breakage in order to get to consolodated builds.  I'm very open to a better approach to that.

What I *do* like about the images_tar line is that it attempts to dynamically figure out the images_tars ... that's a very good trend in our Makefiles :)